cluster-autoscaler/clusterstate/clusterstate.go

```diff
@@ -501,6 +501,14 @@ func (csr *ClusterStateRegistry) IsNodeGroupScalingUp(nodeGroupName string) bool
 	return found
 }
 
+// HasNodeGroupStartedScaleUp returns true if the node group has started scale up.
+// Difference between IsNodeGroupScalingUp() is, this function doesn't check whether
```

Comment:
This comment would be more useful if it explained why this is useful rather than describing how it is implemented.

Reply:
Reworded the comment